Makefile.am

```diff
@@ -39,6 +47,7 @@ EXTRA_DIST = \
 	src/com.endlessm.DiscoveryFeed.in \
 	src/com.endlessm.DiscoveryFeed.src.gresource.xml \
 	data/com.endlessm.DiscoveryFeed.data.gresource.xml \
+	data/com.endlessm.DiscoveryFeed.gschema.xml \
```

Comment:
This looks wrong; the `gschema.xml.in` file should go in `EXTRA_DIST`, while the `gschema.xml` file should go in `CLEANFILES` (that is correct already)

Reply:
Thanks for spotting! https://github.com/endlessm/eos-discovery-feed/pull/79